package/MDAnalysis/topology/PDBParser.py

```diff
@@ -303,6 +308,22 @@ def _parseatoms(self):
         masses = guess_masses(atomtypes)
         attrs.append(Masses(masses, guessed=True))
 
+        # Guessed attributes
+        # Need to pull elements from Atom names
+        # Similar to the check for atomtypes function
+        if not any(elements):
+            # Can further add a check_cg_atom(elements) function in guessers.py to check for CG atom.
+            elements = guess_atom_element(elements)
```

Comment:
`guess_atom_element` guesses from names -- this will return a list of empty strings. It also checks one value at a time. You could use `guess_types` here instead, which just passes it down to `guess_atom_element` anyway.

Reply:
Just adding to @lilyminium's comment. As per #2222, it would be good to throw a user warning when guessing here. That way users are aware of what MDA is doing being the scenes.

Reply:
So, my understanding was that guess_atom_element will guess from a string and I am feeding it the elements string as an input. But I realize that checking the entire string would be the needed option. I can change it to guess_types. 
Also, I added a warning in the following if conditional. Should I shift that a loop above?